reporter/iface.go

```diff
@@ -51,20 +51,27 @@ type TraceReporter interface {
 }
 
 type ExecutableOpener = func() (process.ReadAtCloser, error)
```

Comment:
Let's add comments when introducing exported types/functions, especially when they likely become part of a future public API.